src/module/clearTask.js

```diff
@@ -0,0 +1,10 @@
+import { tasks, storeTasksToLocalStorage } from './taskFunction.js';
+
+const clearAllCompletedTasks = () => {
+  const filteredTasks = tasks.filter((task) => !task.completed);
+  tasks.length = 0;
+  tasks.push(...filteredTasks);
+  storeTasksToLocalStorage();
+};
+
```

Comment:
- [ ] Please fix the "Clear all complete" function as currently upon clicking the `clear all complete` button wrong tasks are being deleted like there are some tasks in the list that are not marked as complete, but they are still being deleted and there are some that are marked complete but not deleted. See the screen cast for more explanation:


https://github.com/axcell-0/todo-list/assets/44798044/27956f73-407a-469c-b2d8-33d15cec580d